modules/emulator/R/minimize.GP.R

```diff
@@ -129,6 +129,7 @@ get.y <- function(gp, xnew, n.of.obs, llik.fn, priors, settings, llik.par = NULL
   
   if(is.null(llik.par)){
     llik.par <- pda.calc.llik.par(settings, n.of.obs, SS)
+    return(llik.par)
```

Comment:
Could you add a comment in the code here explaining why you return from the function here? I'm not really following what you're trying to do, and I do find it an odd design decision to return completely different things depending on whether llik.par is null or not. That also suggests a need to update the roxygen info so that such polymorphism is well documented.  That said, I'm also fairly suspicious that this might not be what you actually want to do in the MCMC.